src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComHresults.cs

```diff
@@ -24,7 +24,7 @@ internal static class ComHresults
 
         internal const int E_NOINTERFACE = unchecked((int)0x80004002);
         internal const int E_FAIL = unchecked((int)0x80004005);
-        internal const int E_NOTIMPL = unchecked((int)0x80000001);
+        internal const int E_NOTIMPL = unchecked((int)0x80004001);
```

Comment:
I think this isn't actually being used and can just be removed.

Reply:
It's not used in the .NET 5 code base, but used in PowerShell. That's how I found it :)